docs/advanced/tls_hardware_acceleration.md

```diff
@@ -127,8 +129,24 @@ To replace a module you have to:
 
 - Provide a header file for your implementation. The file name must be `<Module Name>_alt.h`.
 
-- Set the macro `MBEDTLS_<Module Name>_ALT` to notify mbed TLS and the compiler or linker about the replacement. You can read more on this in the [subsection about setting macros](#How-to-set-the-macros).
+- Set the macro `MBEDTLS_<Module Name>_ALT` to notify mbed TLS and the compiler or linker about the replacement. You can read more on this in the [subsection about setting macros](#how-to-set-the-macros).
 
 ### Where to find the default implementations
 
 The default implementation of the modules are usually in the file `feature/mbedtls/src/<Module Name>.c`. The ECP module is split to two files: `ecp.c` and `ecp_curves.c`.
+
+## Considerations on alternative implementations
+
+### Concurrency
+
+Note that functions in mbed TLS can be called from multiple threads and from multiple processes at the same time. As hardware accelerators are usually a unique resource, it is important to protect all functions against concurrent access.
```

Comment:
“You can call” is not correct: the reader of this document is a developer who implements the additional code in mbed TLS, whereas the caller of the functions is code written by a developer who implements an application that uses the mbed TLS library. Saying that functions “are called” here is common terminology, adding an explicit subject is superfluous. I don't see how to change this sentence to active voice without making it less clear.